packages/app-desktop/gui/NoteEditor/utils/clipboardUtils.test.ts

```diff
@@ -2,6 +2,12 @@ import { htmlToClipboardData } from './clipboardUtils';
 
 describe('clipboardUtils', () => {
 
+	// We are mocking console.error since jsdom throws errors to console when we try to load an invalid img
+	// https://github.com/facebook/jest/pull/5267#issuecomment-356605468
+	beforeEach(() => {
+		console.error = jest.fn();
+	});
```

Comment:
I don't think we are loading any HTML here, just parsing it as text. On which test do you have an error?

In any case it shouldn't be a global "beforeEach", it should be done only in the test that's causing an issue.

Reply:
This test suite actually had the same issue as the new contextMenu test of throwing console.error logs, because `htmlToClipboardData` uses dom elements under the hood to work and 2 tests are feeding it dummy image urls.

Reply:
Ok thanks for clarifying. I wonder if there's a better way to handle this without disabling the console but for now that will do.